spec/system/filters_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Filters' do
+  let(:user) { Fabricate(:user) }
+  let(:filter_title) { 'Filter of fun and games' }
+
+  before { sign_in(user) }
+
+  describe 'Creating a filter' do
+    it 'Populates a new filter from form' do
+      navigate_to_filters
+
+      click_on I18n.t('filters.new.title')
+      fill_in_filter_form
+      expect(page).to have_content(filter_title)
+    end
+  end
+
+  describe 'Editing an existing filter' do
+    let(:new_title) { 'Change title value' }
+
+    before { Fabricate :custom_filter, account: user.account, title: filter_title }
+
+    it 'Updates the saved filter' do
+      navigate_to_filters
+
+      click_on filter_title
+
+      fill_in 'Title', with: new_title
```

Comment:
Shouldn't `Title` be either an identifier (`custom_filter_title`) or use `I18n.t`?